lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
@andy-armstrong What I had in mind were the following best practices:
1. Use `<%-` by default everywhere.  
2. Use `<%=` in exceptional cases where you have something that is already an HTML snippet or you are writing a utility method that needs to handle both (e.g. a user prompt that will display either plain text or HTML snippets).
3. When a variable contains HTML, the best practice would be to name it with a suffix of `Html` as in `titleHtml`.

Your method:
Pro: It is very consistent.
Con: It is very verbose.

My method:
Pro: It makes for generally cleaner looking templates.
Con: It may cause more double-escaping issues without testing, but not more security vulnerabilities.
Con: It is less consistent.

In all cases, the linter will help ensure that people do what we choose.  So, I'm not overly concerned with people having to memorize the various rules.  

I know @cahrens was tending toward consistency in other places.  I'd like to get the thoughts of @cahrens and @nedbat, as well as your response @andy-armstrong. 

Then we'll make a decision.  We need to be quick though, because docs and linter would need to be adjusted if I get convinced to change this.